.github/workflows/dockerfile-health.yml

```
@@ -82,4 +110,11 @@ jobs:
           push: false
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          outputs: type=docker,dest=/tmp/mme_builder.tar
\ No newline at end of file
+          outputs: type=docker,dest=/tmp/mme_builder.tar
+
+  agw_test_devcontainer:
+    uses: maxhbr/magma/.github/workflows/agw-workflow.yml@maxhbr/runAgwWorkflowForNewDevcontainers
```

still my biggest issue: have not yet found a way to reference the `agw-workflow.yml` of the current revision.

---

YOu should be able to suffix the current commit id at the end like it's done here:
`https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#calling-a-reusable-workflow`
THis will look something like 
`maxhbr/magma/.github/workflows/agw-workflow.yml@maxhbr/runAgwWorkflowForNewDevcontainers@FULL_COMMIT_ID`

---

Yes, but I have no access to variables at that point. It is not possible to write something like
```suggestion
    uses: ${{ github.repository }}/.github/workflows/agw-workflow.yml@${{ github.sha }}
```

---

I would say it works but better test it ^^'

---

`have not yet found a way to reference the agw-workflow.yml of the current revision.` Would you like us to ask our friends at the Linux Foundation to relay this question to Github? Is there a documentation / feasibility hole in the new re-use?